utils/test_files_compiler.go

```diff
@@ -0,0 +1,23 @@
+package utils
```

Comment:
Please add a licence header header:
```suggestion
// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See License.txt for license information.

package utils
```